doc/adr/0008-concept-for-handling-ldml.de-in-backend.md

```diff
@@ -0,0 +1,53 @@
+# 6. Concept for handling LDML.de in the backend
+
+Date: 2024-04-12
+
+## Status
+
+Proposed
+
+## Context
+
+We have identified two main issues with our current approach in the backend source code:
+- The current domain model is quite artificial, depicting just one specific use case (a.k.a amending laws).
```

Comment:
Not sure what the "artificial" shall mean, here. 
For me "The current domain model covers only the amending laws use case." is the most relevant information.

Reply:
Btw. while I am fine with mentioning that point, it did not actually occur to me in our discussion, that this was a problem we wanted to solve.

To me it rather was: We found that "amending law" and "target law" were just roles of what are simply "laws".